ServiceDesk/ServiceDesk.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Project naming is inconsistent. Should be `ArcadiaTeamsBot.ServiceDesk`.

Reply:
Projects have been renamed